libcloud/storage/providers.py

```diff
@@ -95,6 +95,10 @@
         "DigitalOceanSpacesStorageDriver",
     ),
     Provider.MINIO: ("libcloud.storage.drivers.minio", "MinIOStorageDriver"),
+    Provider.SCALEWAY: ("libcloud.storage.drivers.scaleway", "ScalewayStorageDriver"),
+    Provider.SCALEWAY_FR_PAR: ("libcloud.storage.drivers.scaleway", "ScalewayFRParStorageDriver"),
```

Comment:
Same as above, those region specific constants should be removed and the main class should take ``region`` constructor argument.